docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,241 @@
+
+You will need:
+
+- An Expo project using SDK 50 or later.
+- **Android TV**:
+  - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For Apple silicon, choose the ARM 64 image. Otherwise, choose the Intel x86_64 image).
+  - After installing the Android TV system image, create an Android TV emulator using that image (the process is the same as creating an Android phone emulator).
+- **Apple TV**:
+  - Xcode 15 or later.
+  - tvOS SDK 17 or later. (This is not installed automatically with Xcode. You can install it later with `xcodebuild -downloadAllPlatforms`).
+
+> **Info** If you are starting with an SDK 49 project, you will need to [upgrade to SDK 50 or higher](/workflow/upgrading-expo-sdk-walkthrough/) before proceeding through the following steps.
+
+## Limitations
+
+Certain Expo tools are not yet supported at this time, most notably:
+
+- `expo-dev-client`
+- Expo Router
+
+<Step label="1">
+
+## Modify dependencies for TV
+
+In **package.json**, modify the `react-native` dependency to use the TV repo, and exclude this dependency from [`npx expo install` version validation](/more/expo-cli/#configuring-dependency-validation).
+
+{/* prettier-ignore */}
+```json package.json
+{
+  /* @hide ... your existing package.json */ /* @end */
+  "dependencies": {
+    /* @hide ... your existing dependencies */ /* @end */
+    "react-native": "npm:react-native-tvos",
+    /* @hide ... */ /* @end */
+  },
+  "expo": {
+    "install": {
+      "exclude": [
+        "react-native"
+      ]
+    }
+  }
+}
+```
+</Step>
+
+<Step label="2">
+
+## Add the TV config plugin
+
+<Terminal cmd={['npx expo install @react-native-tvos/config-tv']} />
+
+Installing the library adds the `@react-native-tvos/config-tv` plugin to your project's **app.json**.
+
+When installed, the plugin will modify the project for TV when either:
+
+- The environment variable `EXPO_TV` is set to `1`
+- The plugin parameter `isTV` is set to `true`
+
+Optionally, set the `showVerboseWarnings` property to `true` to see additional information during prebuild.
+
+```json app.json
+{
+  "plugins": [
+    [
+      "@react-native-tvos/config-tv",
+      {
+        "showVerboseWarnings": true
+      }
+    ]
+  ]
+}
+```
+
+</Step>
+
+<Step label="3">
+
+## Run prebuild
+
+Set the `EXPO_TV` environment variable, and run prebuild to make the TV modifications to the project.
+
+<Terminal cmd={['export EXPO_TV=1', 'npx expo prebuild --clean']} />
+
+> **Note**: The `--clean` argument is recommended, and is required if you have existing Android and iOS directories in the project.
+
+</Step>
+
+<Step label="4">
+
+## Build for Apple TV
+
+After running prebuild, run the following command to build and run the app on an Apple TV simulator:
+
+<Terminal
+  cmd={[
+    'npx expo run:ios'
+  ]}
+/>
+
+</Step>
+
```

Comment:
```suggestion
```

Let's remove this duplicated step since the platform order has now changed.

![CleanShot 2024-01-16 at 13 35 35@2x](https://github.com/expo/expo/assets/10234615/429af9fd-f335-4671-8bd1-0bc0f7ad6f65)